### PR TITLE
M3-5918: Remove `babel` and `webpack` following use of `tsup`

### PR DESCRIPTION
**Pull request by bnussman**
## Description

- Remove unused dependencies and clean up code in the api-v4 and validation packages following #8484

## How to test

- Ensure `@linode/api-v4` and `@linode/validation` still build and work properly in local development and in our ci pipelines

### CODE REVIEW

File: packages/api-v4/package.json [hunk 1]
```diff
@@ -39,54 +39,36 @@
   "unpkg": "./lib/iife/index.js",
   "dependencies": {
     "@linode/validation": "*",
-    "@types/yup": "^0.29.13",
     "axios": "~0.21.4",
     "ipaddr.js": "^2.0.0",
-    "querystring": "^0.2.0",
```
Comment (bnussman) [inline packages/api-v4/package.json:45]: `querystring` was not used anywhere so I removed it

File: packages/api-v4/package.json [hunk 2]
```diff
@@ -39,54 +39,36 @@
   "unpkg": "./lib/iife/index.js",
   "dependencies": {
     "@linode/validation": "*",
-    "@types/yup": "^0.29.13",
```
Comment (bnussman) [inline packages/api-v4/package.json:42]: I moved this types dependency to `devDependencies`

File: packages/api-v4/package.json [hunk 3]
```diff
@@ -100,5 +82,19 @@
     ".{ts,tsx}": [
       "tsc -p tsconfig.json --noEmit true --emitDeclarationOnly false"
     ]
+  },
+  "jest": {
```
Comment (bnussman) [inline packages/api-v4/package.json:86]: I moved the jest config into the `package.json` to match how Cloud Manager has it

**Review by DevDW — APPROVED** [review]
Looks to be good to go to me, probably just need to merge `develop` in to resolve that `yarn.lock` conflict

File: packages/api-v4/package.json (hunk 1 above)
Comment (HanaXu) [reply packages/api-v4/package.json:45]: @bnussman It seems to be used in `session.ts` and `linodeCreate.reducer.ts`

Comment (bnussman) [reply packages/api-v4/package.json:45]: This is the `api-v4` package.json so packages in `manager` should be okay

**Review by HanaXu — APPROVED** [review]

**Pull request merged**